core.js

```diff
@@ -1165,13 +1168,17 @@ class FileTypeParser {
 			};
 		}
 
-		if (
-			this.check([0xFE, 0xFF, 0, 60, 0, 63, 0, 120, 0, 109, 0, 108]) // UTF-16-BOM-LE
-			|| this.check([0xFF, 0xFE, 60, 0, 63, 0, 120, 0, 109, 0, 108, 0]) // UTF-16-BOM-LE
-		) {
+		if (this.check([0xFE, 0xFF])) { // UTF-16-BOM-LE
```

Comment:
This is used in multiple places, so instead of a comment, I would make a `UTF16_BOM_LE` constant.